### PR TITLE
Fixes some plaques descriptions

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Fixes commemorative and memorial plaques descriptions.
Converts all varedited plaques into proper items.
## Why It's Good For The Game

Broken descriptions are broken.
plaque_fix

## Testing

- Compiled and verified both plaques descriptions
## Changelog
:cl:
spellcheck: Fixed commemorative and memorial plaques descriptions
/:cl:

### CODE REVIEW

File: code/game/turfs/simulated/floor/plasteel_floor.dm [hunk 1]
```diff
@@ -25,9 +25,24 @@
...
+	name = "Memorial Plaque"
+	desc = "\"This is a plaque in honour of those who died in the great space lube airlock incident.\" Scratched in beneath that is a crude image of a clown and a spaceman. The spaceman is slipping. The clown is laughing."
+
+/turf/simulated/floor/plasteel/goonplaque/commission
+	name = "Commission Plaque"
+	desc = "Epsilon Eridani Sector - 'Meta' Class Outpost - Commissioned 11/03/2557 - NSS Cerebron"
+
+/turf/simulated/floor/plasteel/goonplaque/nosey
+	name = "Nosey little bastard aren't you?"
+	desc = "Nosey little bastard aren't you?"
```
Comment (S34NW) [inline code/game/turfs/simulated/floor/plasteel_floor.dm:42]: my easter egg, exposed to all!

File: _maps/map_files/Delta/delta.dmm [hunk 2]
```diff
@@ -14790,11 +14790,8 @@
 /obj/machinery/atmospherics/pipe/manifold/hidden/scrubbers{
 	dir = 4
 	},
-/turf/simulated/floor/plasteel{
-	desc = "\"This is a plaque in honour of our comrades on the G4407 Stations. Hopefully TG4407 model can live up to your fame and fortune.\" Scratched in beneath that is a crude image of a meteor and a spaceman. The spaceman is laughing. The meteor is exploding.";
-	dir = 4;
-	icon_state = "plaque";
-	name = "Comemmorative Plaque"
+/turf/simulated/floor/plasteel/goonplaque{
+	dir = 4
```
Comment (S34NW) [inline _maps/map_files/Delta/delta.dmm:14794]: no need for the dir

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 3]
```diff
@@ -53938,7 +53930,7 @@
 /area/toxins/explab)
 "eQf" = (
 /obj/structure/table/wood,
-/obj/structure/sign/kiddieplaque{
+/obj/structure/sign/kiddieplaque/perfect_drone{
 	desc = "A guide to the drone shell dispenser, detailing the constructive and destructive applications of modern repair drones, as well as the development of the incorruptible cyborg servants of tomorrow, available today.";
```
Comment (S34NW) [inline _maps/map_files/MetaStation/MetaStation.dmm:53934]: you didnt remove the var

File: code/game/turfs/simulated/floor/plasteel_floor.dm (hunk 1 above)
Comment (Henri215) [reply code/game/turfs/simulated/floor/plasteel_floor.dm:42]: I tried to find it without cheating, failed.

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 3 above)
Comment (Henri215) [reply _maps/map_files/MetaStation/MetaStation.dmm:53934]: Has been removed now :D

File: _maps/map_files/Delta/delta.dmm (hunk 2 above)
Comment (Henri215) [reply _maps/map_files/Delta/delta.dmm:14794]: Removed

**Review by S34NW — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**